src/lib/styles/themes/dark.scss

```diff
@@ -10,6 +10,10 @@
   --text-text: var(--purple-900);
   --input-focus-background: var(--purple-dark-450);
   --button-primary: var(--purple-dark-accent);
+  --table-background: var(--purple-dark-550);
+  --table-header-background: var(--purple-dark-450);
+  --table-row-hover: var(--purple-dark-500);
+  --table-row: var(--purple-dark-550);
```

Comment:
Optionally you could point this one to the other one like this:
```
--table-row: var(--table-background);
```
But we can also wait with that until we add more layers.

Reply:
I'd rather wait to add layers.

And those should probably be set by the designer.